fn/func.go

```diff
@@ -30,6 +30,27 @@ func Map[I, O any, S []I](s S, f func(I) O) []O {
 	return output
 }
 
+// Keys returns a slice containing all the keys of the given map.
+func Keys[T any, K comparable](s map[K]T) []K {
```

Comment:
This and the below are in an `x` package in the std lib: https://pkg.go.dev/golang.org/x/exp/maps#Values

Reply:
Ah yeah, I keep forgetting. Cool, removed this whole commit.